coda-src/venus/venusutil.cc

```diff
@@ -130,8 +130,8 @@ static const char *VFSOpsNameTemplate[NVFSOPS] = {
     "Statfs",
     "Store",
     "Release",
-    "No-Op",
     "AccessIntent",
+    "No-Op",
     "No-Op"
```

Comment:
why 2 no-ops?

Reply:
Because  `#define NVFSOPS 40` in `venusstats.h`